holder.yml

```diff
@@ -56,60 +56,108 @@ paths:
           description: invalid input!
         "500":
           description: error!
-  /presentations/available:
+  /exchanges/{exchange-id}:
```

Comment:
please define exchange id as a proper parameter, here is an example:

https://github.com/dgarcia360/openapi-boilerplate/blob/master/src/parameters/path/petId.yaml

please include an example as well.